packages/expo-updates/android/src/main/java/expo/modules/updates/EnabledUpdatesController.kt

```diff
@@ -70,18 +69,11 @@ class EnabledUpdatesController(
   private var startupStartTimeMillis: Long? = null
   private var startupEndTimeMillis: Long? = null
 
-  override var shouldEmitJsEvents = false
-    set(value) {
-      field = value
-      UpdatesUtils.sendQueuedEventsToAppContext(value, appContext, logger)
-    }
-
   @Synchronized
   private fun onStartupProcedureFinished() {
     isStartupFinished = true
     startupEndTimeMillis = System.currentTimeMillis()
     (this@EnabledUpdatesController as java.lang.Object).notify()
-    UpdatesUtils.sendQueuedEventsToAppContext(shouldEmitJsEvents, appContext, logger)
```

Comment:
@Kudo - does this sound reasonable to you? My understanding is that there's no longer a need to send the queued events until `OnStartObserving` is called (which sets shouldEmitJsEvents to true and then sends the queued events)

Reply:
yes, it looks correct to me. i forgot why i added it here. let's try to remove the line. thanks for catching up this.

Reply:
I think it's a remnant from a previous world. This line used to be what did all the emitting. Then you added the observer subscribe function and now I think this is obsolete.